agent/php_user_instrument_wraprec_hash.h

```diff
@@ -0,0 +1,249 @@
+/*
+ * Copyright 2022 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#if LOOKUP_METHOD == LOOKUP_USE_WRAPREC_HASHMAP
+
+/* This hashmap will store pointers to wraprecs stored in the linked list.
+ * Since wraprecs are persistent (they're not destroyed between requests), 
```

Comment:
This comment probably needs an update to be more accurate - non-transient wraprecs are persistent. Transient wraprecs are destroyed in `nr_php_remove_transient_user_instrumentation` called from `nr_php_post_deactivate` at the end of request.

Reply:
Updated in [237ca07](https://github.com/newrelic/newrelic-php-agent/pull/587/commits/237ca0707c2b5335f562a9c0ed0ce58f5b4ed0bc)